jetty-home/pom.xml

```diff
@@ -655,6 +655,11 @@
       <artifactId>websocket-jetty-server</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty.websocket</groupId>
+      <artifactId>websocket-jetty-client</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
Why is this added to `jetty-home`? 
I think we always want `websocket-jetty-client` to be in webapps.

Reply:
Do we?  Isn't the point of the module for the server to provide it?
Sure we want to support it in webapps as well, but always?

Reply:
Yes, I think we always want the Jetty specific WebSocket client to be in webapps.
No classloader issue, no issues when deploying (e.g. for testing) in other containers, etc.
There is no point adding it to `jetty-home`, but then cannot be seen by webapps and the server does not use it.